client/src/views/flatEarth.js

```diff
@@ -18,8 +18,15 @@ var flatEarth = function(event) {
         circle.style.width = circle.style.height;
       }
       circle.style.borderRadius = '50%';
+      circle.style.backgroundImage = "";
+    }
+  } else if (event.keyCode === 115) {
+    console.log("here");
```

Comment:
Could you get rid of the console.log()? Otherwise this is all great, I like it.

Reply:
done, sorry about that